client/src/components/Button2/index.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const buttonStyle = {
```

Comment:
Reuse button 1 here, create 1 json and update values that are different in other.